src/components/card/Productcard.jsx

```diff
@@ -59,11 +60,12 @@ const Productcard = ({ product }) => {
           ) : (
             <button
               className="card-btn gh-btn gh-btn-primary"
-              onClick={() =>
+              onClick={() => {
                 isLogged
                   ? cartDispatch({ type: "ADD_TO_CART", payload: product })
-                  : navigate("/login")
-              }
+                  : navigate("/login");
+                toast.success("Added to Cart");
+              }}
```

Comment:
Add to cart toast and Add to wishlist toast appears even when the user isn't logged in.

Reply:
fixed